modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -1,36 +1,35 @@
 // Module included in the following assemblies:
 //
 // * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+
 :_content-type: PROCEDURE
 [id="odc-creating-a-binding-connection-between-components_{context}"]
 = Creating a binding connection between components
 
-You can establish a binding connection with Operator-backed components.
+You can create a binding connection between components, as demonstrated in the following example that uses a PostgresSQL database service and a Spring PetClinic sample application.
 
-This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
-The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+To create a binding connection with a service that is backed by the Crunchy Postgres for Kubernetes, add the Crunchy Postgres for Kubernetes Operator to the *OperatorHub*, and install the Operator. The Crunchy Postgres for Kubernetes Operator provides the PostgresCluster resource that you can use to deploy a PostgreSQL database and expose the binding information in secrets, config maps, status, and spec attributes.
 
 .Prerequisites
 
-* You have created and deployed a Spring PetClinic sample application by using the *Developer* perspective.
-* You have installed {servicebinding-title} from the OperatorHub.
-* You have installed the *Crunchy Postgres for Kubernetes* Operator from the OperatorHub by using the `v5` *Update* channel.
-* You have created and deployed a Crunchy PostgreSQL database instance by using the *Developer* perspective. This instance has the following components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and `hippo-pgbouncer`.
+* You created and deployed a Spring PetClinic sample application in the *Developer* perspective.
```

Comment:
I couldn't find a Spring PetClinic sample application the **Developer** perspective in OpenShift that was provisioned using the Cluster Bot.
Is this https://github.com/devfile-samples/devfile-sample-java-springboot-basic.git or something else? Is there some link, for the reader to verify that the reader installs the correct app?
I also tried to google it, and these are the results that I got:
https://github.com/redhat-developer-demos/spring-petclinic
https://github.com/spring-projects/spring-petclinic
https://developers.redhat.com/quickstarts/eap/spring-petclinic#
https://redhat-developer-demos.github.io/spring-petclinic/
I don't know which one is right for me.
Could you include the correct link on `link:...[Spring PetClinic sample application]` here?

Reply:
@max-cx There is a link to the Getting Started section under the Additional resources section. You will find the YAML for the application with the latest image link (from quay.io) there.

No need to add the link in the additional resources section.

Reply:
Keeping it as is.